screens/HomeScreen.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react';
+import { StyleSheet, Text, TouchableOpacity, View, ScrollView, Image, Button } from 'react-native';
+import { useSelector } from 'react-redux';
+import colors from '../assets/colors/colors';
+import Coin from '../components/Coin';
+import { Crypto } from '../interfaces/types';
+import { RootState } from '../store';
+
+const HomeScreen: FC = ({ navigation }: any) => {
+
+  const list = useSelector((state: RootState) => state.cripto.cryptoList)
+
+  return (
+    <ScrollView style={styles.homeContainer}>
+
+      {/* Header */}
+
+      <View style={styles.headerContainer}>
+        <Text style={styles.headerTitle}>CryptoTracker Pro</Text>
+        <Image source={require('../assets/images/girl.png')} />
```

Comment:
[minor] Preferably import assets using ES6 syntax (import/export)

Reply:
You might want to create an index exporting all assets used on your app, in order to import all necessary assets from the index file